apps/www/components/Frame/Header.js

```diff
@@ -82,8 +82,10 @@ const Header = ({
   }
 
   useEffect(() => {
-    router.prefetch('/meine-republik')
-  }, [])
+    router.prefetch(me ? '/meine-republik' : '/anmelden')
+  }, [me?.id])
```

Comment:
Shouldn't this be just:
```suggestion
  }, [me])
```
?